spec/features/foods/index_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'Foods Index' do
+  describe 'List of Foods from Search' do
+    it 'displays attributes of foods from search' do
+      visit root_path
+
+      fill_in :q, with: 'sweet potatoes'
+      click_button "Search"
+
+      expect(current_path).to eq(foods_path)
+
+      expect(page).to have_content("Data on sweet potatoes")
+      expect(page).to have_content("Total number of items for sweet potatoes is: 45635")
+
+      sweet_potatoes_list = find("#ten_food_results").all("div")
+      expect(sweet_potatoes_list.size).to eq(10)
+
+      within("#ten_food_results") do
+        expect(page).to have_content("GTIN/UPC code: 492111402857")
+        expect(page).to have_content("Description: SWEET POTATOES")
+        expect(page).to have_content("Brand Owner: ARCHER FARMS")
+        expect(page).to have_content("Ingredient(s): SWEET POTATOES.")
+      end
```

Comment:
I like that you're targeting a specific part of your page to expect what's on it. You could also target multiple parts of your page for the different food results. You could do this by making a dynamic class id, maybe you could add the food id to the class name, as to help target which element of the page you're testing.